README.md

```diff
@@ -1,5 +1,16 @@
-# docs-community
+# Python Docs WG
```

Comment:
The title is "Python Docs WG", but the full name below is given as the "PSF Documentation Working Group", and the shortname as "Docs WG". Can we be consistant with the fullname and shortname, and use either one or the other here?

Reply:
Technically, the WG is a formal group of ~four people, who can vote and do other formal things.
Arguably the group isn't working all that well, and IMO it should only be invoked when we need something like a formal pronouncement on some drawn-out argument.

"Python Documentation Community" (or whatever we'll call ourselves) is the wider group of basically anyone who wants to help (including members of the Docs WG). That should be the main focus of this repo.